src/services/api/auth/changePassword.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+import { getCorrectBaseUrl } from "../../../utils/baseUrlConverter";
+
+import { setPasswordEndpoint } from "./apiConstants";
+
+export const changePassword = async (payload: { oldPassword: string; newPassword: string }) => {
```

Comment:
{ oldPassword: string; newPassword: string }

can you create separate type for request props and put it src/types/api/PersonalAccount.ts file?